farm.py

```diff
@@ -0,0 +1,383 @@
+import statistics as stats
+import functions as functions
+
+class Cuerdas:
```

Comment:
is this like, a plot of land? kind of feels like this should be the "Farm" class